requirements.txt

```diff
@@ -9,6 +9,7 @@ werkzeug
 # Optional
 psycopg2
 sqlalchemy
+twine
```

Comment:
Since we only need this for pushing to PyPI, let's not put this in the `requirements.txt`.

We could instead either:

1. Error in `scripts/publish` if the binary file doesn't exist.
2. `pip install twine` in `scripts/publish` if the binary file doesn't exist.